integration-tests/jiva/node-stickiness/jiva_pvc_resource.yaml

```diff
@@ -0,0 +1,12 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: jiva-vol1-1r-claim
+  namespace: jiva-test
+spec:
+  storageClassName: jiva-single-replica
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 5G
```

Comment:
Same comment as below.

Reply:
Enhanced `pkg/kubernetes/persistentvolumeclaim/v1alpha1` to have builder methods.